net/dae/Makefile

```diff
@@ -5,12 +5,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=dae
-PKG_VERSION:=0.8.0
+PKG_VERSION:=0.9.0
```

Comment:
add `GO_PKG_EXCLUDES:=control/kern/tests` below `GO_PKG` to disable broken tests.

Reply:
and `-DMAX_MATCH_SET_LEN=64` should be changed to `-DMAX_MATCH_SET_LEN=1024` in favor of upstream update